core/src/lib.rs

```diff
@@ -192,17 +192,25 @@ pub enum RigidBody {
     /// It is well suited for terrain and static obstacles.
     Static,
 
-    /// A kinematic body is not moved by the physics engine. But it can have user-defined velocity.
+    /// A kinematic body is not moved by the physics engine. But it can have user-defined position.
```

Comment:
```suggestion
    /// A kinematic body is not moved by the physics engine. But it can have user-defined velocity.
```

This was actually correct. heron has an explicit support for velocities on kinematic bodies (that rapier doesn''t have).